ui/src/app/views/data-page/component.tsx

```diff
@@ -0,0 +1,253 @@
+import {Component} from '@angular/core';
+
+import * as React from 'react';
+
+import {CardButton, TabButton} from 'app/components/buttons';
+import {FadeBox} from 'app/components/containers';
+import {ClrIcon} from 'app/components/icons';
+import {SpinnerOverlay} from 'app/components/spinners';
+import {WorkspaceData} from 'app/resolvers/workspace';
+import {cohortsApi, conceptsApi, conceptSetsApi} from 'app/services/swagger-fetch-clients';
+import {ReactWrapperBase, withCurrentWorkspace} from 'app/utils/index';
```

Comment:
You can remove 'index' to shorten the path. I only mention it because my IntelliJ suggested it.